### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by giacomo892**
Substitute #55 . PR to master this time

### CODE REVIEW

**Review by DusKing1 — APPROVED** [review]

**Comment by giacomo892** [comment]
Hi @AlkaMotors can this be merged?

**Comment by AlkaMotors** [comment]
Sorry i didn't get a chance to really look it over. It looks like all the comments for other input types are removed?  I see lots of line changed where nothing appears to be changed at all?

**Comment by giacomo892** [comment]
@AlkaMotors yeah and those are removing of white space at the end of the line

**Comment by cruwaller** [comment]
> @AlkaMotors yeah and those are removing of white space at the end of the line

you also removed newlines at the end of the files. You should keep those.

**Comment by giacomo892** [comment]
> > @AlkaMotors yeah and those are removing of white space at the end of the line
> 
> you also removed newlines at the end of the files. You should keep those.

Can you point out where I did it? Cannot see it

File: Src/dshot.c [hunk 1]
```diff
@@ -238,7 +238,4 @@ for (int i = 15; i >= 9 ; i--){
 #endif
 
 
-}
-
-
-
+}
```
Comment (cruwaller) [inline Src/dshot.c:241]: Keep new line

File: Src/phaseouts.c [hunk 2]
```diff
@@ -311,4 +311,4 @@ void allpwm(){                        // for stepper_sine
 	phaseAPWM();
 	phaseBPWM();
 	phaseCPWM();
-}
+}
```
Comment (cruwaller) [inline Src/phaseouts.c:314]: Keep new line

File: Src/signal.c [hunk 3]
```diff
@@ -125,25 +125,4 @@ if(!armed){
...
-
-
-
-
-
-
-
-
-
+}
```
Comment (cruwaller) [inline Src/signal.c:128]: Keep new line

File: Src/sounds.c [hunk 4]
```diff
@@ -256,4 +256,4 @@ void playBeaconTune3(){
 	TIM1->PSC = 0;
 	signaltimeout = 0;
 	__enable_irq();
-}
+}
```
Comment (cruwaller) [inline Src/sounds.c:259]: Keep new line

**Review by cruwaller — CHANGES_REQUESTED** [review]
@AlkaMotors to decide if he wants to keep the commented protocols just in case.
These changes are quite minor things after all...

**Comment by giacomo892** [comment]
@cruwaller Done. The code needs mayor cleanup, we should start from something :)

**Comment by AlkaMotors** [comment]
I agree...  It does need cleanup, in small steps so I can understand it!   The code for those protocols are gone so we can get rid of the commented lines now.

**Pull request merged**